sdk/cmake/set_wine_module.cmake

```diff
@@ -0,0 +1,21 @@
+
+# FIXME: CORE-5743
+
+function(set_wine_module TARGET)
+    include_directories(BEFORE
+        ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine
+        ${REACTOS_BINARY_DIR}/sdk/include/reactos/wine)
+endfunction()
+
+# FIXME: Replace this call with set_wine_module
+function(set_wine_module_FIXME_1 TARGET)
+    target_compile_definitions(${TARGET} PRIVATE __WINESRC__)
+endfunction()
+
+# FIXME: Replace this call with set_wine_module
+function(set_wine_module_FIXME_2 TARGET)
```

Comment:
Out of curiosity, why did you want to test with these two versions "FIXME_1" and "FIXME_2" ?

Reply:
Read https://github.com/reactos/reactos/pull/7800#discussion_r2002781105